webextensions/api/storage.json

```diff
@@ -372,7 +373,7 @@
               "firefox_android": "mirror",
               "opera": "mirror",
               "safari": {
-                "version_added": false
+                "version_added": "16.4"
```

Comment:
Does Safari support the setAccessLevel method? See https://bugzilla.mozilla.org/show_bug.cgi?id=1724754#c1 and https://github.com/mdn/content/pull/24873

Reply:
@jdatapple are you able to confirm?

Reply:
Safari does not support the setAccessLevel method.